test/parallel/test-net-pipe-connect-errors.js

```diff
@@ -34,7 +35,7 @@ let emptyTxt;
 if (common.isWindows) {
   // on Win, common.PIPE will be a named pipe, so we use an existing empty
   // file instead
-  emptyTxt = path.join(common.fixturesDir, 'empty.txt');
+  emptyTxt = path.join(fixtures.fixturesDir, 'empty.txt');
```

Comment:
This should use `fixtures.path` instead of just replacing where the fixturesDir is coming from.

Reply:
@BridgeAR Updated.  Thanks!